site/topics/output/esap_alu_ram_output.dig

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Swapped the module out and reordered control signals 
![image](https://github.com/user-attachments/assets/8cd47f2e-f79c-4459-b24a-2e8c9c7436ec)